curriculum/challenges/english/10-coding-interview-prep/rosetta-code/top-rank-per-group.md

```diff
@@ -46,7 +46,7 @@ Given the following data:
 
 one could rank the top-rated movie in each genre by calling
 
-`topRankPerGroup(1, data, 'genre', 'rating')`
+`topRankPerGroup(1, testData2, 'genre', 'rating')`
 
 The function should return an array with an array for each group containing the top `n` objects.
 
```

Comment:
```suggestion

For example, given data:
<pre>[
  { name: 'Claire Buckman', id: 'E39876', salary: 27800, dept: 'D101' },
  { name: 'Rich Holcomb', id: 'E01234', salary: 49500, dept: 'D050' },
  { name: 'David Motsinger', id: 'E27002', salary: 19250, dept: 'D050' },
  { name: 'Tim Sampair', id: 'E03033', salary: 27000, dept: 'D101' },
  { name: 'Kim Arlich', id: 'E10001', salary: 57000, dept: 'D050' },
  { name: 'Timothy Grove', id: 'E16398', salary: 29900, dept: 'D101' }
];</pre>
Top two ranking employees in each department by salary would be:
<pre>[ [ { name: 'Kim Arlich', id: 'E10001', salary: 57000, dept: 'D050' },
    { name: 'Rich Holcomb', id: 'E01234', salary: 49500, dept: 'D050' } ],
  [ { name: 'Timothy Grove', id: 'E16398', salary: 29900, dept: 'D101' },
    { name: 'Claire Buckman', id: 'E39876', salary: 27800, dept: 'D101' } ] ]</pre>

```
Perhaps something like this? I've changed data, so it couldn't be used directly as a result for a test.

Reply:
I'd be good with this. @moT01 what do you think?

Reply:
Shouldn't these be in triple back ticks instead of `pre` tags?

Reply:
Yes, they should. I completely overlooked that.